### ISSUE TITLE
@angular folders filed with incorrect .map files

### ISSUE THREAD

**Issue by jffraisse-fa** [angular/angular#8418]
Hey,

When installing @angular dependencies and building with webpack i've got many warnings caused by incorrect .map files in every folders project.

example (@angular/compiler/compiler.js.map):

`{"version":3,"file":"compiler.js","sourceRoot":"/Users/iminar/Dev/angular/modules/@angular/compiler/","sources":["compiler.ts"],"names":[],"mappings":";;;;AAIE;;;;EAIC;AACH,wCAAoC,sCAAsC,CAAC;AAAnE,gFAAmE;AAC3E,yBA2BO,gBAAgB,CAAC;AA1BtB,2DAAkB;AAClB,6DAAmB;AACnB,mDAAc;AACd,6CAAW;AACX,6CAAW;AACX,+EAA4B;AAC5B,qFAA+B;AAC/B,6BAAG;AACH,+CAAY;AACZ,yDAAiB;AACjB,+CAAY;AACZ,+CAAY;AACZ,iGAAqC;AACrC,qDAAe;AACf,iFAA6B;AAC7B,qEAAuB;AACvB,yEAAyB;AACzB,6EAA2B;AAC3B,qEAAuB;AACvB,mEAAsB;AACtB,+DAAoB;AACpB,6DAAmB;AACnB,+DAAoB;AACpB,qEAAuB;AACvB,uEAAwB;AACxB,6DACsB;AAExB,iBAAc,oBAAoB,CAAC,EAAA;AACnC,iBAAc,kBAAkB,CAAC,EAAA"}`

angular version 2.0.0-rc.0,
webpack 1.12.14

**Comment by nimaen** [comment]
Hi,

You can temporary *fix* this by adding a post installation script which do something like this :

```
rm -R node_modules/@angular/**/*.js.map
sed -i '$ d' node_modules/@angular/**/*.js
```

It removes all sourcemaps and all last lines in js files, which should refer to sourcemap files.

**Comment by ericmartinezr** [comment]
Could this be fixed by https://github.com/angular/angular/pull/8410 ?

**Comment by nimaen** [comment]
It should fix it indeed.

**Comment by IgorMinar** [comment]
fixed by 905f38acb814dd5f9ac53563a6acd1dd30433246
